docs/recipes/precompiling-with-webpack.md

```diff
@@ -49,7 +48,7 @@ You can now run `$ ava _build/test.js` to run the tests contained in this output
 
 ### Multiple test files
 
-Things are a little more complicated with multiple test files. We recommend [using babel-register](babelrc.md) until the performance penalty becomes too great.
+Things are a little more complicated with multiple test files. We recommend [using babel-register](babel.md) until the performance penalty becomes too great.
```

Comment:
This should be `@babel/register`. It'd be neat to link to the [right anchor](https://github.com/avajs/ava/blob/master/docs/recipes/babel.md#compile-sources).